dask_expr/_collection.py

```diff
@@ -1622,13 +1639,26 @@ def nunique(self, dropna=True):
         else:
             return uniqs.size
 
-    def drop_duplicates(self, ignore_index=False, split_every=None, split_out=True):
+    def drop_duplicates(
+        self, ignore_index=False, split_every=None, split_out=True, keep="first"
```

Comment:
Curious why keep=None in DataFrame and keep="first" in Series?

Reply:
that was a mistake, thanks for catching